docs(refactor): tutorials MTA-1809 (#443)
scaleway/docs-content: tutorials/host-gitlab-runner/index.mdx

  h1: Hosting your own GitHub runner on an Instance
  paragraph: In this tutorial you will learn how to host your own GitHub runner on an Instance
tags: compute Github-runner Instances 
categories: 
  - compute
dates:
  validation: 2022-02-28
  posted: 2018-04-06
---

[GitHub Actions](https://github.com/features/actions) is a flexible tool that makes it easy to automate all your software workflows. It provides you with an integrated CI/CD that automates nearly anything related to your repository. 

> GitHub provides a limited range of free resources to build your applications through GitHub Actions, and professional developers can reach their limits quickly. Besides, teams working on professional projects prefer having complete control over their build environment. GitHub provides the possibility to execute runners on self-managed Instances. A runner is an application that runs a job from a GitHub Actions workflow.

Reviewer: ```suggestion
GitHub provides a limited range of free resources to build your applications through GitHub Actions, and professional developers can reach these limits quickly. Teams working on professional projects may also prefer to have complete control over their build environment. GitHub provides the possibility to execute runners on self-managed Instances. A runner is an application that runs a job from a GitHub Actions workflow.
```
